Xls/Reader/excel_openpyxl.py

```diff
@@ -25,7 +25,11 @@ def run(argv):
     for row in sheet.iter_rows(row_offset=int(args.start) - 1):
         current_row_read = []
         for cell in row:
-            current_row_read.append(cell.value.encode('utf-8'))
+            value = cell.value
+            if isinstance(value, (int, long)):
```

Comment:
How did you reproduce it? )
Lib is open-sourced - ok. But you`re running on your/our servers which have the latest python versions )
If the last is true, then the code above looks like overcheck.